### ISSUE TITLE
syslog rfc3164 message is malformed

### ISSUE THREAD

**Issue by pvnovarese** [docker/docker#26394]
The message output from the syslog driver using --log-opt syslog-format=rfc3164 has a space between the PRI and HEADER parts of the syslog message, which shouldn't be there.

**Output of `docker version`:**

```
docker@syslog0:~$ docker version
Client:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 17:52:38 2016
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.1
 API version:  1.24
 Go version:   go1.6.3
 Git commit:   23cf638
 Built:        Thu Aug 18 17:52:38 2016
 OS/Arch:      linux/amd64
```

**Output of `docker info`:**

```
docker@syslog0:~$ docker info
Containers: 7
 Running: 0
 Paused: 0
 Stopped: 7
Images: 1
Server Version: 1.12.1
Storage Driver: aufs
 Root Dir: /mnt/sda1/var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 15
 Dirperm1 Supported: true
Logging Driver: syslog
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: bridge host null overlay
Swarm: inactive
Runtimes: runc
Default Runtime: runc
Security Options: seccomp
Kernel Version: 4.4.17-boot2docker
Operating System: Boot2Docker 1.12.1 (TCL 7.2); HEAD : ef7d0b4 - Thu Aug 18 21:18:06 UTC 2016
OSType: linux
Architecture: x86_64
CPUs: 1
Total Memory: 995.8 MiB
Name: syslog0
ID: CMNA:NLJM:VSCO:2QLL:NN5M:6OZY:LEVD:25RH:FFTR:PEAT:W62N:7CB6
Docker Root Dir: /mnt/sda1/var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
Insecure Registries:
 127.0.0.0/8
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

docker host is a docker machine boot2docker instance running in kvm on my fedora host.

logging host is fedora with sysklogd-1.5-19.fc23.x86_64

**Steps to reproduce the issue:**

Docker host:
```
docker@syslog0:~$ sudo docker daemon --log-driver=syslog --log-opt syslog-format
=rfc3164 --log-opt syslog-address=udp://192.168.42.1 &
```

Logging host:
```
pvn@gyarados /home/pvn> sudo syslogd -h -r &
pvn@gyarados /home/pvn> sudo nc -u -l 514
```

Docker host:
```
docker@syslog0:~$ docker run busybox ps
PID USER TIME COMMAND
1 root 0:00 ps
```

Output observed on logging host:
```
pvn@gyarados /home/pvn> sudo nc -u -l 514
<30> Sep 7 20:47:30 syslog0 docker/8b7c83576c56[2635]: PID USER TIME COMMAND
<30> Sep 7 20:47:30 syslog0 docker/8b7c83576c56[2635]: 1 root 0:00 ps
```

RFC3164 is pretty clear that there MUST be a space after the timestamp but does NOT specify need for a space after the priority

works as expected with rfc5424

```
docker@syslog0:~$ sudo docker daemon --log-driver=syslog --log-opt syslog-format
=rfc5424 --log-opt syslog-address=udp://192.168.42.1 &

docker@syslog0:~$ docker run busybox ps
PID USER TIME COMMAND
1 root 0:00 ps
```

```
pvn@gyarados /home/pvn> sudo nc -u -l 514
110 <30>1 2016-09-07T21:19:09Z syslog0 docker/81256921c9e8 3408 docker/81256921c9e8 PID USER TIME COMMAND
105 <30>1 2016-09-07T21:19:09Z syslog0 docker/81256921c9e8 3408 docker/81256921c9e8 1 root 0:00 ps
```

**Describe the results you received:**

```
<30> Sep 7 20:47:30 syslog0 docker/8b7c83576c56[2635]: PID USER TIME COMMAND
<30> Sep 7 20:47:30 syslog0 docker/8b7c83576c56[2635]: 1 root 0:00 ps
```

**Describe the results you expected:**

```
<30>Sep 7 20:47:30 syslog0 docker/8b7c83576c56[2635]: PID USER TIME COMMAND
<30>Sep 7 20:47:30 syslog0 docker/8b7c83576c56[2635]: 1 root 0:00 ps
```

**Additional information you deem important (e.g. issue happens only occasionally):**

I looked at daemon/logger/syslog/syslog.go and I think this may actually be a bug in the go syslog package but ... I'm not sure?

**Comment by cpuguy83** [comment]
Thanks, looks like this is already reported in RackSec/srslog#15

**Comment by cpuguy83** [comment]
I've made a PR to fix this: RackSec/srslog#17
